pkg/local_object_storage/metabase/control.go

```diff
@@ -128,6 +128,13 @@ func (db *DB) init(reset bool) error {
 	})
 }
 
+// SyncCounters forces to synchronize the object counters.
+func (db *DB) SyncCounters() error {
+	return db.boltDB.Update(func(tx *bbolt.Tx) error {
+		return syncCounter(tx, db.epochState.CurrentEpoch(), true)
```

Comment:
We shouldn't block in an `Update` transaction, could you move epoch fetching out of it?

Reply:
fixed